src/components/JobConfigurationForBulkScheduler.vue

```diff
@@ -0,0 +1,212 @@
+      <ion-modal class="date-time-modal" :is-open="isOpen" @didDismiss="() => isOpen = false">
+        <ion-content force-overscroll="false">
+          <ion-datetime 
+            show-default-buttons 
+            hour-cycle="h23" 
+            :value="job.setTime ? getDateTime(job.setTime) : ''" 
+            @ionChange="updateRunTime($event, job)" 
+          />
+        </ion-content>
+      </ion-modal>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Schedule") }}</ion-label>
+      <ion-select :interface-options="customPopoverOptions" :value="job.frequency" interface="popover" :placeholder='$t("Bulk schedule")' @ionChange='setFrequency($event, job)'>
+        <ion-select-option v-for="freq in generateFrequencyOptions" :key="freq.value" :value="freq.value">{{ $t(freq.label) }}</ion-select-option>
+      </ion-select>
+    </ion-item>
+    <div class="actions">
+        <ion-button size="small" fill="outline" color="danger" @click="removeJob(job.jobId)">
+          <ion-icon slot="start" :icon="closeOutline"/>
+          {{ $t("Remove") }}
+        </ion-button>
+    </div>
+  </ion-card>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import {
+  IonBadge,
+  IonButton,
+  IonCard,
+  IonContent,
+  IonDatetime,
+  IonIcon,
+  IonItem,
+  IonItemDivider,
+  IonLabel,
+  IonNote,
+  IonModal,
+  IonSelect,
+  IonSelectOption,
+} from "@ionic/vue";
+import {
+  calendarClearOutline,
+  flashOutline,
+  copyOutline,
+  timeOutline,
+  timerOutline,
+  syncOutline,
+  personCircleOutline,
+  pinOutline,
+  closeOutline
+} from "ionicons/icons";
+import { mapGetters, useStore } from "vuex";
+import { useRouter } from "vue-router";
+import { handleDateTimeInput, showToast } from "@/utils";
+import { DateTime } from 'luxon';
+import { translate } from '@/i18n'
+
+export default defineComponent({
+  name: "JobConfigurationForBulkScheduler",
+  components: {
+    IonBadge,
+    IonButton,
+    IonCard,
+    IonContent,
+    IonDatetime,
+    IonIcon,
+    IonItem,
+    IonItemDivider,
+    IonLabel,
+    IonModal,
+    IonNote,
+    IonSelect,
+    IonSelectOption,
+  },
+  data() {
+    return {
+      isOpen: false,
+    }
+  },
+  props: ["job", "selectedShopifyConfigs", "selectedEComStoreId"],
+  computed: {
+    ...mapGetters({
+      bulkJobs: 'job/getBulkJobs',
+      userProfile: 'user/getUserProfile',
+      shopifyConfigs: 'user/getShopifyConfigs',
+    }),
+    getEComStoreName() {
+      return this.userProfile.stores.find((store: any) => store.productStoreId === this.selectedEComStoreId).storeName;
+    },
+    getShopifyConfigNames() {
+      // find matching shopifyConfig objects and return their names 
+      return this.shopifyConfigs.filter((config: any) => this.selectedShopifyConfigs.includes(config.shopId)).map((config: any) => config.name).join(', ');
+    },
+    generateFrequencyOptions(): any {
+      const optionDefault = [{
```

Comment:
What if we are scheduling a job in which we need to give slow frequency options to select? Like if we have a job `abc` in which we only allow slow freq options, but in bulk scheduling case it will always have default freq options.

Reply:
Yes, sir. Proceeding as per the discussion.